text-ui-test/input.txt

```diff
@@ -8,6 +8,15 @@ add parallel c 20
 calc power
 exit
 summary
+<<<<<<< HEAD
```

Comment:
Seems to be an unresolved merge conflict in this file. Might want to edit this file.

Reply:
Sure, will remove it.